specification/transport_layer/transport_layer.tex

```diff
@@ -379,7 +386,11 @@ \subsection{Redundant interface support}
 utilization rates of the redundant interfaces and different phasing of their traffic;
 however, that is not an issue for UAVCAN.
 If perfectly simultaneous frame submission is not possible, interfaces with lower numerical index
-should be handled in the first order.
+should be handled in the first order\footnote{It should be kept in mind that redundant interfaces
+are used for increased fault tolerance, not for load sharing reasons.
+Whenever a node is connected to an interface, the likelihood of its failure is increased;
```

Comment:
Comma splice and ambiguity in the predicate. Perhaps: Whenever a node is connected to an interface the likelihood of the interface failing is increased